storage/test/storage/ArrayStorageTest.java

```diff
@@ -0,0 +1,93 @@
+package storage;
+
+import model.Resume;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import storage.exception.ResumeAlreadyStoredException;
+import storage.exception.ResumeNotFoundException;
+
+abstract class ArrayStorageTest {
+    protected static final int MAX_TEST_STORAGE_SIZE = 4;
+
+    private String uuidDevCV = "ssj-34eh32";
+    private String uuidManagerCV = "ewo-232hsj";
+    private String uuidAdminCV = "qxb-238sdf";
+
+    Resume devCV = new Resume(uuidDevCV, "Developer-CV");
+    Resume managerCV = new Resume(uuidManagerCV, "Manager-CV");
+    Resume adminCV = new Resume(uuidAdminCV, "Admin-CV");
```

Comment:
I guess it can be added to `@BeforeEach` method. There is only `devCv` that shared once. Can be created using `new` one more time. It's not a big problem as tests should be isolated as much as possible.

Reply:
Done